src/main/java/com/mraof/minestuck/skaianet/SkaianetHandler.java

```diff
@@ -343,7 +343,7 @@ private void closeConnection(SburbConnection connection, ISburbComputer clientCo
 	
 	public void requestInfo(ServerPlayer player, PlayerIdentifier p1)
 	{
-		checkData();
+		checkData(); //TODO checkData is called again(conditionally) in requestInfo, is there a way to reduce redundancy?
```

Comment:
I do not feel like I can make an adequate judgement on this. This comment continues the TODO in ComputerReference, will remove it if someone has a comment to make in this PR

Reply:
Reworking `checkData()` would proably be out of scope for this PR, so I suggest leaving it be for now.

Reply:
Would you like me to remove the TODO here or is it appropriate to keep?

Reply:
As long as there's already a todo adressing `checkData()` elsewhere, I'd say remove this one